### PR TITLE
fix compilation on windows

### PR DESCRIPTION
**Pull request by lasconic**

### CODE REVIEW

File: thirdparty/poppler/goo/glibc.cc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//========================================================================
+//
+// glibc.h
```
Comment (Jojo-Schmitz) [inline thirdparty/poppler/goo/glibc.cc]: typo in comment?

File: thirdparty/poppler/goo/glibc.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+//========================================================================
+
+#ifndef GLIBC_H
+#define GLIBC_H
+
+#include "config.h"
+
+#include <time.h>
+
+extern "C" {
```
Comment (Jojo-Schmitz) [inline thirdparty/poppler/goo/glibc.h]: but glibc.cc is C++ code, isn't it?

**Comment by lasconic** [comment]
The code is from poppler

**Comment by Jojo-Schmitz** [comment]
I know (or at least guessed) but that doesn't make it error free ;-)

**Pull request merged**